*** tests/A-13-21-M-ok.out	2022-11-03 22:36:23.340977000 +0000
--- tests/A-13-21-M-ok.outhyp	2022-11-07 12:59:39.471663769 +0000
***************
*** 101,116 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: VOICE|1|969001|969006|13|260|FINISHED
! VOICE|2|969001|969014|13|260|FINISHED
! VIDEO|3|969011|969014|1|20|FINISHED
  VIDEO|4|969011|969015|1|20|FINISHED
  VIDEO|5|969011|969014|1|20|FINISHED
  VIDEO|6|969011|969015|1|20|FINISHED
  VIDEO|7|969011|969014|2|40|FINISHED
- TEXT|8|969011|969015|50|4|FINISHED
- VOICE|9|969011|969015|1|10|FINISHED
- TEXT|10|969011|969015|120|4|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 101,113 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969011|969015|50|4|FINISHED
! VOICE|2|969011|969015|1|10|FINISHED
! TEXT|3|969011|969015|120|4|FINISHED
  VIDEO|4|969011|969015|1|20|FINISHED
  VIDEO|5|969011|969014|1|20|FINISHED
  VIDEO|6|969011|969015|1|20|FINISHED
  VIDEO|7|969011|969014|2|40|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
